README.md

```diff
@@ -1,73 +1,144 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
+<!-- Improved compatibility of back to top link: See: https://github.com/othneildrew/Best-README-Template/pull/73 -->
+<a name="readme-top"></a>
+<!--
+*** Thanks for checking out the Best-README-Template. If you have a suggestion
+*** that would make this better, please fork the repo and create a pull request
+*** or simply open an issue with the tag "enhancement".
+*** Don't forget to give the project a star!
+*** Thanks again! Now go create something AMAZING! :D
+-->
 
-# Project Name
 
-> Description the project.
 
+<!-- PROJECT SHIELDS -->
+<!--
+*** I'm using markdown "reference style" links for readability.
+*** Reference links are enclosed in brackets [ ] instead of parentheses ( ).
+*** See the bottom of this document for the declaration of the reference variables
+*** for contributors-url, forks-url, etc. This is an optional, concise syntax you may use.
+*** https://www.markdownguide.org/basic-syntax/#reference-style-links
+-->
+[![LinkedIn][linkedin-shield]][linkedin-url]
 
-## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
 
-## Live Demo (if available)
+<!-- PROJECT LOGO -->
+<br />
+<div align="center">
+  <a href="https://github.com/github_username/repo_name">
+    <img src="https://github.com/sireclifford/marvelmetrics/blob/6c1350b67a46fa7259290cae23f662abe12dd3e2/metric-app-full.png" alt="Logo" width="400" height="400" />
+  </a>
 
-[Live Demo Link](https://livedemo.com)
+<h3 align="center">Metrics App</h3>
 
+  <p align="center">
+    It's about building a mobile web application to check a list of metrics (numeric values) that you will create making use of React and Redux.
+    <br />
+    <br />
+    <a href="https://github.com/github_username/repo_name">View Demo</a>
```

Comment:
- [ ] Please ensure that you deploy the Project using Heroku, Netlify or GitHub Pages and everything is working perfectly in the deployment. Kindly also add the deployment link in the ReadMe as is a Requirement for this project.

Reply:
I has been done